pandas/plotting/_core.py

```diff
@@ -3188,19 +3188,85 @@ def kde(self, bw_method=None, ind=None, **kwds):
 
     def area(self, x=None, y=None, **kwds):
         """
-        Area plot
+        Draw a stacked area plot.
+
+        An area plot displays quantitative data visually.
+        This function wraps the matplotlib area function.
 
         Parameters
         ----------
-        x, y : label or position, optional
-            Coordinates for each point.
-        `**kwds` : optional
+        x : label or position, optional
+            Index, coordinates for X axis.
```

Comment:
The "Index, " part is a bit confusing I think. I would reword this as : `Coordinates for the Y axis. By default uses the index.`